lib/matplotlib/widgets.py

```diff
@@ -2031,8 +2039,8 @@ def on_select(min: float, max: float) -> Any
         The direction along which to draw the span selector.
 
     minspan : float, default: 0
-        If selection is less than or equal to *minspan*, do not call
-        *onselect*.
+        If selection is less than or equal to *minspan*, the selection is
+        removed (when already existing) or not completed.
```

Comment:
I would perhaps write `or cancelled` (I think "cancelled" reads better than "not completed", but don't feel very strongly about this).

Reply:
Yes, cancelled is better, I was not very happy with "completed" neither.